openarc/util.c

```diff
@@ -891,9 +892,9 @@ arcf_mkarray(char *in)
 	if (out == NULL)
 		return (const char **) NULL;
 
-	for (p = strtok(in, ","); *p != '\0'; p = strtok(NULL, ","))
-		out[c] = p;
-	out[c] = NULL;
+	for (p = strtok_r(in, ",", &ctx); p != NULL; p = strtok_r(NULL, ",", &ctx))
+		out[c++] = p;
+	out[n] = NULL;
```

Comment:
No, scratch that, I forgot that `n` is modified higher up. It's just a pointless change, not actually wrong.